spec/docx/smoke/api_document_spec.rb

```diff
@@ -169,4 +169,89 @@
+      expect(docx.elements.first.nonempty_runs[0].formula_run[0].numerator.formula_run[0].text).to eq('dx')
+      expect(docx.elements.first.nonempty_runs[0].formula_run[0].denominator.formula_run[0].text).to eq('dy')
+    end
+
+    it 'Check with sFormat="latex"' do
+      expect(docx.elements.first.nonempty_runs[2].class).to eq(OoxmlParser::DocxFormula)
+      expect(docx.elements.first.nonempty_runs[2].formula_run[0].text).to eq('dx/dy')
+    end
+  end
+
+  describe 'ApiDocument | GetCurrentWord method' do
+    let(:docx) { builder.build_and_parse('js/docx/smoke/api_document/get_current_word.js') }
+
+    # TODO: 'check after release'
+    before { skip('New methods') if builder.semver < Semantic::Version.new('8.2.0') }
+
+    it 'Check without sWordPart' do
+      expect(docx.elements[1].nonempty_runs.first.text).to eq('word is GetCurrentWord')
+    end
+
+    it 'Check with sWordPart="before"' do
+      expect(docx.elements[2].nonempty_runs.first.text).to eq('before cursor: GetCur')
+    end
+
+    it 'Check with sWordPart="after"' do
+      expect(docx.elements[3].nonempty_runs.first.text).to eq('after cursor: rentWord')
+    end
+  end
+
+  describe 'ApiDocument | ReplaceCurrentWord method' do
+    let(:docx) { builder.build_and_parse('js/docx/smoke/api_document/replace_current_word.js') }
+
+    # TODO: 'check after release'
+    before { skip('New methods') if builder.semver < Semantic::Version.new('8.2.0') }
+
+    it 'Check without sWordPart' do
+      expect(docx.elements.first.nonempty_runs[1].text).to eq('replace ')
+    end
+
+    it 'Check with sWordPart="before"/"after"' do
+      expect(docx.elements.first.nonempty_runs[2].text).to eq('current word')
+    end
+  end
+
+  it 'ApiDocument | SelectCurrentWord method' do
+    # TODO: 'check after release'
+    skip('New methods') if builder.semver < Semantic::Version.new('8.2.0')
+    docx = builder.build_and_parse('js/docx/smoke/api_document/select_current_word.js')
+    expect(docx.elements[1].nonempty_runs[0].text).to eq('selected text is SelectCurrentWord')
+  end
+
+  describe 'ApiDocument | GetCurrentSentence method' do
+    let(:docx) { builder.build_and_parse('js/docx/smoke/api_document/get_current_sentence.js') }
+
+    # TODO: 'check after release'
+    before { skip('New methods') if builder.semver < Semantic::Version.new('8.2.0') }
+
+    it 'Check without sWordPart' do
+      expect(docx.elements[1].nonempty_runs.first.text).to eq('sentence is For check GetCurrentSentence.')
+    end
+
+    it 'Check with sWordPart="before"' do
+      expect(docx.elements[2].nonempty_runs.first.text).to eq('before cursor: For check ')
+    end
+
+    it 'Check with sWordPart="after"' do
+      expect(docx.elements[3].nonempty_runs.first.text).to eq('after cursor: GetCurrentSentence.')
+    end
+  end
+
+  it 'ApiDocument | ReplaceCurrentSentence method' do
+    # TODO: 'check after release'
+    skip('New methods') if builder.semver < Semantic::Version.new('8.2.0')
```

Comment:
I think it's better to write not just skip, but `skip('New methods for DocBuilder v8.2.0)`

Because current variant will be not quite clear when you just run from CLI all the tests - you will see a lot of `New methods` but need to debug further for which version it's relevant